op-challenger/metrics/metrics.go

```diff
@@ -195,6 +207,15 @@ func (m *Metrics) RecordGamesStatus(inProgress, defenderWon, challengerWon int)
 	m.trackedGames.WithLabelValues("challenger_won").Set(float64(challengerWon))
 }
 
+func (m *Metrics) RecordActedL1Block(n uint64) {
+	m.bnMutex.Lock()
+	defer m.bnMutex.Unlock()
+	if m.highestActedL1BlockNumber < n {
+		m.highestActedL1BlockNumber = n
+		m.highestActedL1Block.Set(float64(n))
+	}
```

Comment:
We should be able to remove this logic now and just set the metric.  The coordinator will always be providing the correct value and only increment.